osu.Game.Rulesets.Taiko/Difficulty/Skills/Stamina.cs

```diff
@@ -1,16 +1,48 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-using System.Linq;
 using osu.Game.Rulesets.Difficulty.Preprocessing;
 using osu.Game.Rulesets.Difficulty.Skills;
-using osu.Game.Rulesets.Difficulty.Utils;
 using osu.Game.Rulesets.Mods;
 using osu.Game.Rulesets.Taiko.Difficulty.Preprocessing;
 using osu.Game.Rulesets.Taiko.Objects;
 
 namespace osu.Game.Rulesets.Taiko.Difficulty.Skills
 {
+    /// <summary>
+    /// Stamina of a single key, calculated based on repetition speed.
+    /// </summary>
+    public class SingleKeyStamina
+    {
+        private double previousHitTime;
+
+        /// <summary>
+        /// Similar to <see cref="StrainDecaySkill.StrainValueOf(DifficultyHitObject)"/>
+        /// </summary>
+        public double StrainValueOf(DifficultyHitObject current)
+        {
+            if (previousHitTime == -1)
```

Comment:
Uh... this condition will never be hit now after the latest commit removed the initialisation of this to -1. What I intended to suggest with my previous review comment is to do this:

```csharp
double? previousHitTime;

// in StrainValueOf():
if (previousHitTime == null)
{
    previousHitTime = current.StartTime;
    return 0;
}
// rest of code follows
```

Reply:
Apologies, was in a bit of a rush and forgot to initiate a `double?`